bucket/wireshark.json

```
@@ -31,6 +31,11 @@
     "suggest": {
         "nmap": "nmap"
```

```suggestion
         "pcap": "nmap"
```

Maybe you can also change the suggest as we only need npcap in nmap. This is more clear.